news.xml

```diff
@@ -2,7 +2,7 @@
 ---
 
 <?xml version="1.0" encoding="utf-8"?>
-<feed xmlns="http://www.w3.org/2005/Atom">
+<feed xmlns="https://www.w3.org/2005/Atom">
```

Comment:
Please see my [previous comment](https://github.com/fortran-lang/fortran-lang.org/pull/228#issuecomment-798884529) regarding this.